modules/imgcodecs/include/opencv2/imgcodecs/macosx.h

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
To resolve problem with ABI checker on Linux builder please add this platform-specific header into skip list [here](https://github.com/opencv/opencv/blob/76be3529f491aa5835602021c65ef99975837cac/cmake/templates/opencv_abi.xml.in#L34).

Reply:
I was having trouble getting rid of the ABI error, but I overlooked this file. Thank you for pointing it out! 🙇‍♂️